### PR TITLE
Add android runtime

### PR DESCRIPTION
**Pull request by Ma-Dan**
Basic android demo.

### CODE REVIEW

File: runtime/android/app/src/main/cpp/kaldi-native-fbank/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+add_subdirectory(csrc)
```
Comment (csukuangfj) [inline runtime/android/app/src/main/cpp/kaldi-native-fbank/CMakeLists.txt:1]: Is it possible to use `cmake/kaldi-native-fbank.cmake` to download it during building?

File: runtime/android/app/src/main/cpp/sherpa-ncnn/csrc/lstm-model.cc [hunk 2]
```diff
@@ -0,0 +1,125 @@
+/**
```
Comment (csukuangfj) [inline runtime/android/app/src/main/cpp/sherpa-ncnn/csrc/lstm-model.cc:1]: Can we reuse the code from `sherpa-ncnn/csrc/lstm-model.cc`?

Comment (Ma-Dan) [reply runtime/android/app/src/main/cpp/sherpa-ncnn/csrc/lstm-model.cc:1]: Yes, maybe we can soft link to these source files.

File: runtime/android/app/src/main/cpp/kaldi-native-fbank/CMakeLists.txt (hunk 1 above)
Comment (Ma-Dan) [reply runtime/android/app/src/main/cpp/kaldi-native-fbank/CMakeLists.txt:1]: Yes, I will try ExternalProject_Add function to download and process kaldi-native-fbank project.

File: runtime/android/app/src/main/cpp/sherpa-ncnn/csrc/lstm-model.cc (hunk 2 above)
Comment (Ma-Dan) [reply runtime/android/app/src/main/cpp/sherpa-ncnn/csrc/lstm-model.cc]: Fixed

File: runtime/android/app/src/main/cpp/kaldi-native-fbank/CMakeLists.txt (hunk 1 above)
Comment (Ma-Dan) [reply runtime/android/app/src/main/cpp/kaldi-native-fbank/CMakeLists.txt]: Fixed

**Comment by csukuangfj** [comment]
By the way, I just fixed the building issues of kaldi-native-fbank for android in
https://github.com/csukuangfj/kaldi-native-fbank/pull/3/files

Please use kaldi-native-fbank v1.7

**Comment by pingfengluo** [comment]
@Ma-Dan Nice work! How about the RTF?

**Pull request closed without merging**